File: Bilibili_Innocent_Lab/app/src/test/java/com/bilibili/pegasus/data/base/BasePegasusData.java
package com.bilibili.pegasus.data.base;

import com.bilibili.app.comm.list.common.api.model.PlayerArgs;

public class BasePegasusData {
    public String getBizType() { return "UGC"; }
    public String getCardType() { return "CARD_TYPE_AV"; }
    public Object getAdInfo() { return null; }
    public String getCardGoto() { return "av"; }
    public String getGoTo() { return "av"; }
    public String getUri() { return "bilibili://video/test"; }
    public String getParam() { return "BV1TEST"; }
    public String getTitle() { return "title"; }
    public String getSubtitle() { return "subtitle"; }
    public String getDesc() { return "desc"; }
    public PlayerArgs getPlayerArgs() { return new PlayerArgs(); }
}

File: Bilibili_Innocent_Lab/app/src/test/java/com/bilibili/pegasus/data/base/BasePegasusData.kt
package com.bilibili.pegasus.data.base

import com.bilibili.app.comm.list.common.api.model.PlayerArgs

open class BasePegasusData {
    open fun getBizType(): String = "UGC"
    open fun getCardType(): String = "CARD_TYPE_AV"
    open fun getAdInfo(): Object? = null
    open fun getCardGoto(): String = "av"
    open fun getGoTo(): String = "av"
    open fun getUri(): String = "bilibili://video/test"
    open fun getParam(): String = "BV1TEST"
    open fun getTitle(): String = "title"
    open fun getSubtitle(): String = "subtitle"
    open fun getDesc(): String = "desc"
    open fun getPlayerArgs(): PlayerArgs = PlayerArgs()
}
